python/lsst/pipe/tasks/generateEphemerides.py

```diff
@@ -337,17 +336,38 @@ def run(self, visitSummaries, mpcorb, de440s, sb441_n16, obsCodes, linux_p1550p2
                 meta_kernel_file.write(meta_kernel_text)
             self.log.info('Sorcha process begun')
 
+            # FIXME: this is a workaround for breakage in sbpy 0.5.0 caused
```

Comment:
Please file a ticket to remove the workaround, and reference the new ticket here.